plugins/catalog/src/components/CreateComponentButton/indext.ts

```diff
@@ -13,4 +13,5 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
```

Comment:
Why did we rename this file to `indext.ts`? Is this intentional? I've never seen an `indext.ts` file before. 🤔

Reply:
Not intentional - fixed it - good catch! :)